blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -0,0 +1,64 @@
+Activity=Activity
+Artifacts=Artifacts
+Author=Author
+Branch=Branch
+Branches=Branches
+Build=Build
+Changes.by=Changes by {0}
+Changes=Changes
+Changes.condense={0} changes
+Commit=Commit
+Completed=Completed
+Dashboard=Dashboard
+Date=Date
+Date.readable.short=MMM DD h:mma Z
+Date.readable.long=MMM DD YYYY h:mma Z
+Date.duration.format=m[ minutes] s[ seconds]
+Date.duration.hint.format=M [month], d [days], h[h], m[m], s[s]
+Download.artifact=Download the artifact
+Duration=Duration
+Duration.param=Duration {0}
+# the emptyState keys assume that you will parse them with a markdown parser
+EmptyState.activity=# Ready, get set...\nHmm, looks like there are no runs in this pipeline\u2019s history.\n\nCommit to the repository _{0}_ or run the pipeline manually.
+EmptyState.artifacts=There are no artifacts for this pipeline run.\n\n
+EmptyState.branches=# Branch out\nCreate a branch in the repository _{0}_ and Jenkins will start testing your changes.\n\nGive it a try and become a hero to your team.
+EmptyState.branches.notSupported=# Branches are unsupported\nBranch builds only work with the _Multibranch Pipeline_ job type. This is just one of the many reasons to switch to Jenkins Pipeline.\n\n[Learn more](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.changes=There are no changes for this pipeline run.\n\n
+EmptyState.pr.notSupported=# Pull Requests are unsupported\nValidated pull request builds only work with the _Multibranch Pipeline_ job type. This is just one of the many reasons to switch to Jenkins Pipeline.\n\n[Learn more](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.pr=# Push me, pull you\nWhen a Pull Request is opened on the repository _{0}_, Jenkins will test it and report the status of your changes back to the pull request on Github.
+EmptyState.tests=There are no tests run for this build.\n\n
+Enable=Enable
+Failed=Failed {0}
+Failing={0} failing
+Failures=Failures
+Fixed=Fixed
+Health=Health
+Latest.build=Latest Build
+Latest.message=Latest message
+Loading=Loading...
+Message=Message
+More=Show more
+Name=Name
+New.failures=New Failures
+New.Pipeline=New Pipeline
+No.changes=No changes
+noSteps=There are no steps.
+Not.found.heading=Page not found (404)
+Not.found.message=Jenkins could not find the page you were looking for. Check the URL for errors or press the back button.
+Open.dashboard=Open Dashboard
+Passed=Passed {0}
+Passing={0} passing
+Passing.singular=Passing
+Pipeline=Pipeline
+PR=Pull Requests
+Run.now=Run Now
+Serenity=How's the serenity?
+Show.complete.logs=Show complete log
+Size=Size
+Skipped.singular=Skipped
+Skipped=Skipped {0}
+Status=Status
+Summary=Summary
+Tests.passing=All tests are passing
+Tests.passing.count=Nice one! All {0} tests for this pipeline are passing.
+Tests=Tests
```

Comment:
If you see the history of this PR you find @tfennelly requesting the opposite. Having it changed it once already I am not really keen to do it again (at least not in this PR)